_content/papers/vyaghri-jataka-in-mahavastu-and-fobenxingji jing_marciniak-katarzyna.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Vyāghrī-jātaka in the Mahāvastu and Fobenxingji jing"
```

Comment:
Add *s around things to make them italic.

Also: your filename has a space in it